### ISSUE TITLE
Popup boxes hide behind client if always on top is enabled

### ISSUE THREAD

**Issue by Yonder0133** [runelite/runelite#2690]
Was wondering if there was a way to set the Logout Confirmation box that says to have the same frame as the parent, or to have the RuneLite client have an exception where it allows that box to be above it in frame but nothing else? The client overlaps the Logout Confirmation sometimes, not allowing one to log out unless you move the client which I forget often.

**Comment by deathbeam** [comment]
No, because this was about logout. For screen markers, yo should make new issue

**Comment by sadegr** [comment]
@deathbeam then why did you mark #3197 a duplicate of this issue when it's also about screen markers?

**Comment by Abextm** [comment]
It seems I closed this issue in error; #3265 only partially fixes this.

**Comment by russel204** [comment]
Im still having this issue. If Display warning on exit is set to "Logged in" and I have Enable Client Always on top checked if i accidentally click the close button it freezes the client because the pop-up is behind the client. the only fix I have found is to kill the popup with task manager

**Comment by Nightfirecat** [comment]
I could not reproduce this when testing. Both the color picker and the confirm-exit dialog boxes appeared over the client when I had "Always on top" enabled.
Is anyone else able to reproduce this issue?
